### ISSUE TITLE
Ruffle Extension Doesn't Detect Elements Inserted With innerHTML

### ISSUE THREAD

**Issue by Justin-CB** [ruffle-rs/ruffle#462]
On these 3 pages:
https://justinbesterportfolio.000webhostapp.com/DArts/Animate/ghost.php
https://justinbesterportfolio.000webhostapp.com/DArts/Animate/fire.php
https://justinbesterportfolio.000webhostapp.com/DArts/Animate/fire2.php
Flash embed content is added to the HTML document using innerHTML. The ruffle extension fails to detect this, most likely due to the fact that it either doesn't detect dynamically inserted flash embeds at all, or doesn't detect this as dynamically inserting elements.

**Comment by kmeisthax** [comment]
We used to attempt to polyfill dynamic content using `MutationObserver`. It had significant performance problems and broke a lot of modern webapps in the process. Ergo, it was removed pretty quickly.

**Comment by Justin-CB** [comment]
I'll see if I can find another way to do this.

**Comment by Toad06** [comment]
That seems to work if you add this:
```
<script>
	window.RufflePlayer = window.RufflePlayer || {};
	window.RufflePlayer.config = {
		"polyfills": ["static-content", "plugin-detect", "dynamic-content"]
	};
</script>
```

(interesting to see the extension can interoperate with the user's configuration)

**Comment by Justin-CB** [comment]
I've submitted a pull request that hopefully will resolve this issue without creating the issues kmeisthax referenced.

**Comment by Herschel** [comment]
Should be fixed by fa063a6, which enabled the dynamic detection by default -- if there are web apps that break because of the polyfill, this should be filed as a separate issue.
